pom.xml

```diff
@@ -60,7 +60,8 @@
     <main.basedir>${project.basedir}</main.basedir>
     <jee.version>7</jee.version>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <cdap.version>6.1.1</cdap.version>
+    <cdap.version>6.6.0</cdap.version>
```

Comment:
`DATETIME` data type is not available in cdap-api-common 6.1.1.
So, I had to use cdap.version 6.6.0 a1691d4500f8670c38f28b341322dac56d37f279

Reply:
I think datetime support was added in 6.4.0. Generally prefer to depend on the lowest CDAP API version that's supported so that the plugin can be compatible with as many CDAP versions are possible.

Reply:
downgraded from `6.6.0` to `6.4.0` 326fadb444e9fa01ddc710c3ae549590703f264e